src/drive/web/modules/views/OnlyOffice/useConfig.js

```diff
@@ -19,6 +21,25 @@ const useConfig = () => {
   useEffect(
     () => {
       if (!isQueryLoading(queryResult) && fetchStatus !== 'error' && !config) {
+        if (isSharedWithMe(data)) {
+          const {
+            protocol,
+            instance,
+            document_id,
+            subdomain
+          } = data.data.attributes
+
+          const link = generateWebLink({
+            cozyUrl: `${protocol}://${instance}`,
+            pathname: '',
+            hash: `/onlyoffice/${document_id}/fromSharing`,
+            slug: 'drive',
+            subDomainType: subdomain
+          })
+
+          return (window.location = link)
```

Comment:
window.location will open a browser tab when we're using the mobile app. is it wanted?

Reply:
Oh ok je ne savais pas. Qu'en penses-tu @poupotte ? On parle ici de la redirection sur le cozy hôte lors de la consultation d'un doc only office partagé en tant que destinataire.

@Crash-- si jamais, y'a t'il une façon de faire une redirection tout en restant dans l'app ? Est-ce que `window.location.href` a le même comportement ? Concernant le `router` je ne pense pas qu'il permette de sortir de l'instance, si ?

Reply:
@poupotte C'est d'ailleurs un cas à prendre en compte sur les scénarios mobiles

@JF-Cozy Y a moyen de faire des choses en restant dans l'app, mais le plus compliqué c'est de trouver comment y revenir et ça, c'est pas simple. Donc si ça convient en l'état... 🙏

Reply:
vu avec @poupotte on reste sur le comportement actuel :)